### DISCUSSION TITLE
Check service is running

### DISCUSSION THREAD

**Discussion by carlocorradini** [OpenRC/openrc#575] [Q&A] [answered]
There is something similar to `systemctl is-active --quiet <SERVICE>` (<https://unix.stackexchange.com/q/396630/458944>) to check if **SERVICE** is running (exit 0 if running)?
Thanks 🥳

**Comment by powerjungle** [comment, marked as answer]
`rc-service --quiet <SERVICE> status`

Also don't forget, there's: `man openrc` and `man rc-service` and `rc-service --help`
